test/error-placement.js

```diff
@@ -311,6 +311,21 @@ test( "test existing non-error aria-describedby", function( assert ) {
 	strictEqual( "", $("#testForm17text-error").text(), "Error label is empty for valid field" );
 });
 
+test( "test existing non-error aria-describedby with bracket", function( ) {
+	expect( 2 );
+	var form = $( "#testForm18" ),
+		field = $( "#testForm18\\[text\\]");
+
+	field.attr("aria-describedby", "testForm18\\[text\\]-description testForm18\\[text\\]-error");
```

Comment:
I still don't understand this approach. You can call `.valid()` multiple times to verify the change isn't applied multiple times. That way you can also verify that it works properly in the first place.